foundations/javascript_basics/DOM_manipulation_and_events.md

```diff
@@ -386,7 +386,7 @@ You can find a more complete list with explanations of each event on [this page]
 
 Manipulating web pages is the primary benefit of the JavaScript language! These techniques are things that you are likely to be messing with _every day_ as a front-end developer, so let's practice!
 
-1.  Grab the first exercise in Wes Bos's JavaScript30 program by cloning the repo at [https://github.com/wesbos/JavaScript30](https://github.com/wesbos/JavaScript30). Code along with the [Video Tutorial](https://www.youtube.com/watch?v=VuN8qwZoego) to build the rest of the exercise. In the video you will notice that a deprecated [keycode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) keyboard event is used, replace it with the recommended [code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) keyboard event. 
+1.  Grab the first exercise in Wes Bos's JavaScript30 program by cloning the repo at [https://github.com/wesbos/JavaScript30](https://github.com/wesbos/JavaScript30). Code along with the [Video Tutorial](https://www.youtube.com/watch?v=VuN8qwZoego) to build the rest of the exercise. In the video you will notice that a deprecated [keycode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) keyboard event is used, replace it with the recommended [code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) keyboard event. You will also need to replace the `data-key` tags accordingly (e.g. `65` in the [keycode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) event corresponds to `KeyA` in the [code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) event).
```

Comment:
I find this wording slightly confusing and it links to the both resources twice, which I think adds to the confusion. Would it be possible to do something like this:
`... keyboard event and replace the `data-key` tags accordingly.`

Reply:
Thank you for the feedback, it sounds much better.